common-tasks/software-update-vm.md

```diff
@@ -115,19 +115,19 @@ As the TemplateVM is used for creating filesystems for other AppVMs, where you a
 
 There are several ways to deal with this problem:
 
--   Only install packages from trusted sources -- e.g. from the pre-configured Fedora repositories. All those packages are signed by Fedora, and as we expect that at least the package's installation scripts are not malicious. This is enforced by default (at the [firewall VM level](/doc/firewall/)), by not allowing any networking connectivity in the default template VM, except for access to the Fedora repos.
+-   Only install packages from trusted sources -- e.g. from the pre-configured Fedora repositories. All those packages are signed by Fedora, and we expect that at least the package's installation scripts are not malicious. This is enforced by default (at the [firewall VM level](/doc/firewall/)), by not allowing any networking connectivity in the default template VM, except for access to the Fedora repos.
 
 -   Use *standalone VMs* (see below) for installation of untrusted software packages.
 
--   Use multiple templates (see below) for different classes of domains, e.g. a less trusted template, used for creation of less trusted AppVMs, would get various packages from somehow less trusted vendors, while the template used for more trusted AppVMs will only get packages from the standard Fedora repos.
+-   Use multiple templates (see below) for different classes of domains, e.g. a less trusted template, used for creation of less trusted AppVMs, would get various packages from somewhat less trusted vendors, while the template used for more trusted AppVMs will only get packages from the standard Fedora repos.
```

Comment:
Mmm... this one is a bit nuanced I think.

I agree that this use of `somehow` isn't the normal English use that I'm familiar with, but this usage is consistent between ITL people, so I assumed it was a regional Polish-English colloquialism or something. From the other contexts I've seen it, I think the intended meaning may be closer to "potentially" or "possibly" or "in some way" (meaning: not certain to be the case, or certain to be the case but for unknown or unclear reasons) rather than than the suggested "somewhat" (meaning: is the case, only to which degree or extend is unknown or uncertain). It doesn't really matter here, but since there are a bunch of uses of this, perhaps it's worth clarifying?

@marmarek @rootkovska @woju ?